AzureTable/Microsoft.DataTransfer.AzureTable/Sink/TableAPIBulkSinkAdapter.cs

```diff
@@ -149,6 +159,11 @@ private async Task FlushToTableAsync(CancellationToken cancellation)
                                         ex.Message, op[0].Entity.PartitionKey, listofDocumentsNotCommitted), ex
                                 );
                                 exceptions.Add(ex);
+
+                                //Log the failure to a cosmosDB table in  the provided account.
```

Comment:
Format the comment properly. At least this rule should be satisfied: http://stylecop.soyuz5.com/SA1005.html

Also "cosmosDB" doesn't look like a correct branding name, as well as two consecutive spaces.